.github/workflows/weekly.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     if: github.repository_owner == 'apache'
     strategy:
       matrix:
-        flink: [1.16-SNAPSHOT, 1.17-SNAPSHOT, 1.18-SNAPSHOT]
```

Comment:
@MartijnVisser @snuyanzin I was not able to make the nightly e2e tests to run across all three versions (the changes in `ExecutionConfig` are resulting in NPEs all over the place for `1.16-SNAPSHOT`), I had to drop it.

Reply:
Looks reasonable.
The thing i would like to clarify whether it's ok to drop support for 1.16 while it's still maintained in main flink repo?
@MartijnVisser may be you have more info about this ?

Reply:
> May I ask you please to brach off v3.0? I believe I have no permissions to do that

Sure. Do you want it to match the tag of the 3.0 release, or are there more commits that should be in there?

Reply:
I think d40b74ddba3cdaccfb45880aa0080d31dfa1e971 is good branching point (in case we would need to do another release), but it seem like the version should be 1.0.0 (not 3.0.0), and the main will become 2.0.0, is my understanding correct? Thank you.

Reply:
@reta I've created [v1.0 ](https://github.com/apache/flink-connector-opensearch/tree/v1.0) and also set the version for `main` to `2.0-SNAPSHOT`.

Reply:
Thanks a lot @MartijnVisser , rebased, I think we are good to go